spec/aruba/spawn_process_spec.rb

```diff
@@ -56,5 +56,35 @@
 
       it { expect {process.start}.to raise_error Aruba::LaunchError }
     end
+
+    context "with a childprocess launch error" do
+      let(:child) { instance_double(ChildProcess::AbstractProcess) }
```

Comment:
Mmhh... I think we should not mock "Classes" we own. Just make it an double, to an instance one.